src/tools/impl/tinytex.ts

```diff
@@ -335,9 +335,8 @@ This will instruct TeX Live to create symlinks that it needs in <bin_dir_on_path
     const envPath = Deno.env.get("QUARTO_TEXLIVE_BINPATH");
     if (envPath) {
       paths.push(envPath);
-    } else if (Deno.build.os !== "windows") {
-      paths.push(...suggestUserBinPaths());
     } else {
+      paths.push(... await suggestUserBinPaths());
       paths.push(tlmgrPath);
     }
```

Comment:
With this change, `binPathMessage` should be updated also below I think ? 

TBH I am not just why this need to be changed for Windows and if it works effectively.

We may end up with a different configuration when `quarto tools install tinytex` is run from Powershell or CMD, or run from Git bash. In the latter `~/bin` will be in PATH I think, but in the former it won't. So `suggestUserBinPaths()` will filter out from possible paths.

Reply:
I'm fine to revert this. I think there's a larger issue at play here of Quarto acting as a package manager of sorts. It is a huge convenience, but perhaps also a quagmire. That's a discussion for a separate issue, and I got sidetracked here by finding this connection with suggestUserBinPaths.